pyEpiabm/pyEpiabm/tests/test_unit/test_property/test_space_foi.py

```diff
@@ -12,60 +13,85 @@ class TestSpatialInfection(TestPyEpiabm):
     determine whether infection events occur within places.
     Each function should return a number greater than 0.
     """
-    @classmethod
-    def setUpClass(cls) -> None:
+
+    def setUp(self) -> None:
         """Intialise a population with one infector and one
         infectee, both in the same place and household.
         """
-        super(TestSpatialInfection, cls).setUpClass()  # Sets up parameters
-        cls.cell = pe.Cell()
-        cls.cell.add_microcells(1)
-        cls.microcell = cls.cell.microcells[0]
-        cls.microcell.add_people(2)
-        cls.infector = cls.microcell.persons[0]
-        cls.infector.infectiousness = 1.0
-        cls.infectee = cls.microcell.persons[1]
-        cls.time = 1.0
+        super(TestSpatialInfection, self).setUpClass()  # Sets up parameters
+        self._population = pe.Population()
+        self._population.add_cells(1)
+        self._population.cells[0].add_microcells(1)
+        self._population.cells[0].microcells[0].add_people(2)
+        for person in self._population.cells[0].microcells[0].persons:
+            person.update_status(InfectionStatus(7))
+        self._population.cells[0].microcells[0].persons[0].infectiousness = 1.0
+        self.placeclosure = PlaceClosure(
```

Comment:
Similar holds here. Make a separate function to test the place closure and put parameters in there.